asyn/drvAsynSerial/drvAsynIPServerPort.c

```diff
@@ -382,6 +383,7 @@ int createServerSocket(ttyController_t *tty) {
     int i;
     struct sockaddr_in serverAddr;
     int oneVal=1;
+    char srvaddrtxt[50];
```

Comment:
Is there a special reason to use 50 here ?
For IPv4 addresses we typically use 32.
If we want to prepared for IPv6, 64 is a typical value

Reply:
Yes, it is: for an IPv4 address, 23 (or 32) bytes are sufficient. For an IPv6 address you need at least 49 bytes ("[aaaa:bbbb:cccc:dddd:eeee:ffff:gggg:hhhh]:ppppp" plus NUL character), so I used a buffer size of 50.

Reply:
Did you reserve some bytes for the interfaceID ?
The "%eno1" from below:
[fe80::2547:708e:a277:b32c%eno1]